### PR TITLE
Update doc and version

### PR DESCRIPTION
**Pull request by nCastle1**
Enhances clarity around app registration process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,15 @@ If there are changes made in the Original repository, you can sync the fork to k
 3. ```git checkout master``` to checkout your fork's local master branch.
 4. ```git merge upstream/master``` to sync your local master' branch with upstream/master. Note: Your local changes will be retained and your fork's master branch will be in sync with the upstream repository.
 
+#### Configure the app
+
+Maps app uses authenticated ArcGIS Online services for routing and geocoding. You need to configure the app for OAuth authentication.
+
+1. Log in to developers.arcgis.com.
+2. Create a [new application](https://developers.arcgis.com/applications/new).
+3. After creating the application, navigate to the 'Authentication' tab and add a redirect URI. Note the Client ID and Redirect URI.
```
Comment (esreli) [inline README.md:77]: > Note the Client ID and Redirect URI.

At this step, there will not yet be a specified Redirect URI.  A few questions here, does the maps app dot net require a Redirect URI? (The iOS counterpart so I don't know why it wouldn't) If so, is it configurable? And can we improve the README setup instructions to ensure a user knows how to configure the app in this aspect?
```suggestion
3. After creating the application, navigate to the 'Authentication' tab and add a redirect URI. Note the **Client ID** and **Redirect URI**.
```

File: README.md [hunk 2]
```diff
@@ -68,6 +68,15 @@ If there are changes made in the Original repository, you can sync the fork to k
 3. ```git checkout master``` to checkout your fork's local master branch.
 4. ```git merge upstream/master``` to sync your local master' branch with upstream/master. Note: Your local changes will be retained and your fork's master branch will be in sync with the upstream repository.
 
+#### Configure the app
+
+Maps app uses authenticated ArcGIS Online services for routing and geocoding. You need to configure the app for OAuth authentication.
+
+1. Log in to developers.arcgis.com.
```
Comment (esreli) [inline README.md:75]: Can you make this a hyperlink?

File: RELEASE.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## 1.0.8
+
+- Updates doc to make requirements for getting started clearer.
```
Comment (esreli) [inline RELEASE.md:5]:
```suggestion
- Improves app configuration and setup instructions for clarity.
```

**Review by esreli — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -68,14 +69,15 @@ If there are changes made in the Original repository, you can sync the fork to k
...
+### Configure the app
 
 Maps app uses authenticated ArcGIS Online services for routing and geocoding. You need to configure the app for OAuth authentication.
 
-1. Log in to developers.arcgis.com.
+1. Log in to [developers.arcgis.com](https://developers.arcgis.com).
 2. Create a [new application](https://developers.arcgis.com/applications/new).
-3. After creating the application, navigate to the 'Authentication' tab and add a redirect URI. Note the Client ID and Redirect URI.
-4. In **Helpers\Configuration.cs** within the **MapsApp.Shared** project, replace the default client ID and Redirect URI with the values noted from step 3.
+3. After creating the application, navigate to the 'Authentication' tab and add a redirect URI. Any URI should work, but try to choose something unique to your app, like `my-maps-app://auth`.
```
Comment (esreli) [inline README.md:78]: Is `my-maps-app://auth` the default redirect URI? If so, I would mention that it is... this way people can try out the app by adding the default redirect URI specified here.

Comment (nCastle1) [reply README.md]: Default is `https://developers.arcgis.com` in code. I added a remark about that

File: README.md [hunk 5]
```diff
@@ -68,6 +69,16 @@ If there are changes made in the Original repository, you can sync the fork to k
...
 4. ```git merge upstream/master``` to sync your local master' branch with upstream/master. Note: Your local changes will be retained and your fork's master branch will be in sync with the upstream repository.
 
+### Configure the app
+
+Maps app uses authenticated ArcGIS Online services for routing and geocoding. You need to configure the app for OAuth authentication.
+
+1. Log in to [developers.arcgis.com](https://developers.arcgis.com).
+2. Create a [new application](https://developers.arcgis.com/applications/new).
+3. After creating the application, navigate to the 'Authentication' tab and add a redirect URI. Any URI should work, but try to choose something unique to your app, like `my-maps-app://auth`. The default value in the app's code is `https://developers.arcgis.com`, so you can use that as an interim value while evaluating the app.
+4. While on the 'Authentication' page, note the **Client ID** and the **Redirect URI** you specified.
```
Comment (esreli) [inline README.md:79]: I would suggest 'Authentication' is more of a tab than a page, though feel free to ignore this if you disagree.

```suggestion
4. While on the 'Authentication' tab, note the **Client ID** and the **Redirect URI** you specified.
```

File: README.md [hunk 6]
```diff
@@ -68,6 +69,16 @@ If there are changes made in the Original repository, you can sync the fork to k
...
+5. In **Helpers\Configuration.cs** within the **MapsApp.Shared** project, replace the default **client ID** and **Redirect URI** with the values noted earlier.
```
Comment (esreli) [inline README.md:80]: Is **Helpers\Configuration.cs** hyper-linkable?

**Review by esreli — APPROVED** [review]
LGTM 👍

**Pull request merged**